wp-curate.php

```diff
@@ -3,7 +3,7 @@
  * Plugin Name: WP Curate
  * Plugin URI: https://github.com/alleyinteractive/wp-curate
  * Description: Plugin to curate homepages and other landing pages
- * Version: 2.4.0
+ * Version: 2.5.0
```

Comment:
I feel like this could be 2.4.1. Installing it won't break anything.

Reply:
Updated in [0140c29](https://github.com/alleyinteractive/wp-curate/pull/235/commits/0140c296932d93cf1309a469b72f499ae5ff8b89)